cli/utils.py

```diff
@@ -223,6 +224,77 @@ def print_batch_exception(batch_exception):
     log.error("-------------------------------------------")
 
 
+'''
+    Job submission
+'''
+
+def print_jobs(jobs: List[aztk.spark.models.Job]):
+    print_format = '{:<34}| {:<10}| {:<20}'
+    print_format_underline = '{:-<34}|{:-<11}|{:-<21}'
+
+    log.info(print_format.format('Job', 'State', 'Creation Time'))
+    log.info(print_format_underline.format('', '', '', ''))
+    for job in jobs:
+
+        log.info(
+            print_format.format(
+                job.id,
+                job.state,
+                utc_to_local(job.creation_time)
+            )
+        )
+
+
+def print_job(job: aztk.spark.models.Job):
+    print_format = '{:<36}| {:<15}'
+
+    log.info("")
+    log.info("Job             %s", job.id)
+    log.info("------------------------------------------")
+    log.info(print_format.format("State:", job.state))
+    log.info(print_format.format("Transition Time:", utc_to_local(job.state_transition_time)))
+    log.info("")
```

Comment:
Nitpick: This doesn't follow the format for print cluster:
![image](https://user-images.githubusercontent.com/11393045/34743168-b5d00e9a-f53d-11e7-8f87-e24df215af33.png)


vs.

![image](https://user-images.githubusercontent.com/11393045/34743147-a05f020a-f53d-11e7-9092-fa5c9771a6dd.png)